src/node_debug_options.cc

```diff
@@ -77,9 +77,11 @@ bool DebugOptions::ParseOption(const std::string& option) {
     argument = option.substr(pos + 1);
   }
 
+  // Note that --debug-port and --debug-brk are undocumented, but to be
+  // supported until 7.x is no longer supported, not even in LTS (see #12364).
   if (option_name == "--inspect") {
     enable_inspector = true;
-  } else if (option_name == "--inspect-brk") {
+  } else if (option_name == "--inspect-brk" || option_name == "--debug-brk") {
```

Comment:
This is how I made it fail: https://github.com/jkrems/node/commit/50ecfe448af82bfb0710ab69b0dc9da113a00de4

E.g. track `--debug-brk` being used and then fail (hard) if it was used w/o `--inspect`. Because that case would mean someone relatively clearly was trying to start the old protocol.